chat_server/jwt_option.ts

```diff
@@ -0,0 +1,7 @@
+var jwt_option = {
+    "secret": "password",
```

Comment:
put into a config file (and only an example synced with git)

Reply:
What type on config file ? Env/Json/Js ?